assets/css/okta-plugin-a11y.css

```diff
@@ -14,6 +14,12 @@ button {
   background: none;
 }
 
+a:link,
+a:visited,
+a:active {
```

Comment:
Considered changing this in [assets/sass/_base.scss](https://github.com/okta/okta-signin-widget/blob/9897f7a238a04fd5e7b841ec01bbfa431de91bd7/assets/sass/_base.scss#L7) but needs designer verification

Reply:
Running this branch locally, looks like `_base.scss` is overriding this underline, and `text-decoration: none` is still in effect.

Philosophically, I support underlines on links. (Odyssey disagrees. But we're optimizing for different use cases, so divergence could be justifiable.) If we do introduce underlines, we should do so carefully.

For example, we'll need to make sure that buttons built as `<a>` don't underline their labels.

**Adding underlines to links is probably easiest?**

I believe [WCAG guidance](https://www.w3.org/WAI/WCAG21/Techniques/general/G183) is that while not preferred, lack of underlines is acceptable if the link color has 3:1 contrast against surrounding text.

The current color palette in the widget does not achieve that contrast.

The current Odyssey palette [does pass contrast requirements](https://webaim.org/resources/linkcontrastchecker/?fcolor=007BBD&bcolor=767676) against body text, but not for links against subtle text.

I'm open to changing our color use to the Odyssey palette instead of adding underlines, but that's probably harder. Widget links currently in the body color would need to change to the link color, and subtle text next to a link would need to change to the darker body color. There'd be a lot to audit, and I expect a lot of CSS to wrangle with.